### DISCUSSION TITLE
Security Onion Installation Problem

### DISCUSSION THREAD

**Discussion by anik001g** [Security-Onion-Solutions/securityonion#3894] [Unsupported Versions] [answered]
Hi,

I'm not a security Master or Cyber pro. I work for a small ISP company. Recently some incident happened into my network and I want to investigate issues like this and found Security Onion which is perfect solution for me.

But problem is, it took almost 20-24 hour to install (VM config: 26 Cpu 26GB RAM, Storage: 1000GB, interface: 2). After that I got an error "Install had a problem Please see /root/sosetup.log for details. A summary of errors can be found in /root/errors.log. Press OK to exit." I have to monitor almost 1000Mbps traffic but for now I need to monitor 100Mbps. Need expert help and resolve this issue. All info I got attached below for your reference.

[errors.log](https://github.com/Security-Onion-Solutions/securityonion/files/6323229/errors.log)

![error](https://user-images.githubusercontent.com/23041528/114987239-c28a1d80-9eb6-11eb-9070-1e9d72eefbe3.JPG)

[root@SecurityOnion anik]# so-status

Checking Docker status

    Docker ------------------------------------------------------------------------------------ [ OK ]

Checking container statuses

    so-aptcacherng ---------------------------------------------------------------------------- [ OK ]
    so-cortex --------------------------------------------------------------------------------- [ OK ]
    so-curator -------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry ------------------------------------------------------------------------- [ OK ]
    so-elastalert ----------------------------------------------------------------------------- [ OK ]
    so-elasticsearch -------------------------------------------------------------------------- [ OK ]
    so-filebeat ------------------------------------------------------------------------------- [ OK ]
    so-fleet ---------------------------------------------------------------------------------- [ OK ]
    so-grafana -------------------------------------------------------------------------------- [ OK ]
    so-idstools ------------------------------------------------------------------------------- [ OK ]
    so-influxdb ------------------------------------------------------------------------------- [ OK ]
    so-kibana --------------------------------------------------------------------------------- [ OK ]
    so-kratos --------------------------------------------------------------------------------- [ OK ]
    so-logstash ------------------------------------------------------------------------------- [ OK ]
    so-mysql ----------------------------------------------------------------------------- [ MISSING ]
    so-nginx ---------------------------------------------------------------------------------- [ OK ]
    so-playbook ------------------------------------------------------------------------------- [ OK ]
    so-redis ---------------------------------------------------------------------------------- [ OK ]
    so-sensoroni ------------------------------------------------------------------------------ [ OK ]
    so-soc ------------------------------------------------------------------------------------ [ OK ]
    so-soctopus ------------------------------------------------------------------------------- [ OK ]
    so-steno ---------------------------------------------------------------------------------- [ OK ]
    so-strelka-backend ------------------------------------------------------------------------ [ OK ]
    so-strelka-coordinator -------------------------------------------------------------------- [ OK ]
    so-strelka-filestream --------------------------------------------------------------------- [ OK ]
    so-strelka-frontend ----------------------------------------------------------------------- [ OK ]
    so-strelka-gatekeeper --------------------------------------------------------------------- [ OK ]
    so-strelka-manager ------------------------------------------------------------------------ [ OK ]
    so-suricata ------------------------------------------------------------------------------- [ OK ]
    so-telegraf ------------------------------------------------------------------------------- [ OK ]
    so-thehive -------------------------------------------------------------------------------- [ OK ]
    so-thehive-es ----------------------------------------------------------------------------- [ OK ]
    so-wazuh ---------------------------------------------------------------------------------- [ OK ]
    so-zeek ----------------------------------------------------------------------------------- [ OK ]

**Comment by LS3-Allen** [comment, marked as answer]
I had the same issue and solved it by simply starting MySQL.
sudo so-mysql-start

**Reply by anik001g** [reply]
thanks. I found the problem which was hardware related. After rebooting my Physical machine problem solved. Now Security onion is running but my 40 GB RAM usages always full. This is why I'm little bit worried about. Currently I'm monitoring my 100 Mbps live traffic by port mirroring.

**Reply by ggranata14** [reply]
I am having the same error as you, but all of my docker statuses are complete. All you had to do was reboot your physical machine, or were there more steps to it than that?

**Reply by dougburks** [reply]
@ggranata14 From the [discussion guidelines](https://github.com/Security-Onion-Solutions/securityonion/discussions/1720):
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.

**Reply by 3isenHeiM** [reply]
Same here, the command `sudo so-mysql-start` solved it.

**Reply by cyberdaddy93** [reply]
> I had the same issue and solved it by simply starting MySQL. sudo so-mysql-start

Did you redo the setup? after sudo so-mysql-start ? if so how?

**Reply by 3isenHeiM** [reply]
Nope, only the mysql start command.

**Reply by sandraimmaculate** [reply]
Hi, I'm studying on securityonin  I want to download security onion 2.3 in standalone node im also getting the same error. if i go and check in sosetup.log file the error was like (Command failed with exit code 1; will retry in 60 seconds (5 / 5)...
Command continues to fail; giving up.
Could not pull so-grafana:2.3.260, please ensure connectivity to ghcr.io
Errors detected during setup; skipping post-setup steps to allow for analysis of failures.) i also checked for connectivity its good 0% packet loss. when tried yesterday i got same error for wazuh now im getting for grafana.                  i tried executing  the command sudo so-mysql-start it shows command not found    if i type any command like so-status it shows command not found  security onion is not installed i guess  can you please help me with this issues

**Reply by dougburks** [reply]
@sandraimmaculate Please use your separate discussion at https://github.com/Security-Onion-Solutions/securityonion/discussions/10835 rather than continuing this old discussion.

**Comment by 004500sk** [comment]
can any one able to tell, how to reach /root/sosetup.log,
/rrot/errors.log file, please

**Reply by dougburks** [reply]
@004500sk Please start a new discussion rather than continuing this old discussion.
